gmusicapi/clients/musicmanager.py

```diff
@@ -385,7 +385,8 @@ def download_song(self, song_id):
 
     @utils.accept_singleton(basestring)
     @utils.empty_arg_shortcircuit(return_code='{}')
-    def upload(self, filepaths, transcode_quality='320k', enable_matching=False):
```

Comment:
Yeah, I was back-and-forth on that in my head. I went with this because it made sense to group related parameters and relying on implicit keyword argument positioning is considered bad practice (though we've all done it/do it : P), but i can change it.